### PR TITLE
Feature/add testnet docs

### PR DESCRIPTION
**Pull request by erasmus**
- Add testnet section to the docs

### CODE REVIEW

**Comment by alinnow** [comment]
Deploy preview for *satoshipay-api* ready!

Built with commit 5919d996c116ebf9455600b219600d231dfc6325

https://deploy-preview-35--satoshipay-api.netlify.com

File: source/includes/_testnet.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<script src="https://wallet.satoshipay.io/testnet/satoshipay.js"></script>
+```
+
+In order to use the testnet version of the [widget](#the-satoshipay-widget) on your web page, call the javascript below:
+
+`https://wallet.satoshipay.io/testnet/satoshipay.js`
+
+## Testnet Funds
+
+In order to get testnet funds, find a Stellar friendbot which disperses free testnet lumens to a testnet account. Solar wallet is free and comes with a friendbot feature for testnet. Download the wallet on [solarwallet.io](https://solarwallet.io/).
```
Comment (alinnow) [inline source/includes/_testnet.md]:
```suggestion
In order to get testnet funds, find a Stellar friendbot which sends free testnet lumens to a testnet account. Solar wallet is free and comes with a friendbot feature for testnet. Download the wallet from [solarwallet.io](https://solarwallet.io/).
```

Comment (alinnow) [reply source/includes/_testnet.md]: I think you meant _disburse_, but I think that's not a common term

**Comment by erasmus** [comment]
@alanpearce thanks for review and the correction. Made an additional wording fix, mainly for aesthetics.  Please merge when accepted.

**Review by alinnow — APPROVED** [review]

**Pull request merged**